packages/git/git.pacscript

```diff
@@ -1,17 +1,17 @@
 name="git"
-pkgname="git"
-version="2.31.1"
-url="https://github.com/git/git/archive/refs/tags/v2.31.1.tar.gz"
+version="2.32.0"
+url="https://www.kernel.org/pub/software/scm/$name/$name-$version.tar.gz"
 build_depends="dh-autoreconf"
 depends="libcurl4-gnutls-dev libexpat1-dev \
   gettext libz-dev libssl-dev asciidoc xmlto docbook2x install-info"
-breaks=""
 replace="git"
 description="git is a fast, scalable, distributed revision control system with an unusually rich command set that provides both high-level operations and full access to internals."
-hash="b1c0e95e9861b5d1b9ad3d8deaa2d8c7f02304ffc1b5e8869dd9fb98f9a0d436"
+hash="6038f06d396ba9dab2eee541c7db6e7f9f847f181ec62f3d8441893f8c469398"
 maintainer="WRM-42 <y8bsbahy@anonaddy.me>"
 prepare() {
+          sudo mkdir -p /usr/src/pacstall/$name/usr/share/licenses/$name
```

Comment:
```suggestion
```
Not needed as `install` automatically creates the directories

Reply:
Done